docs/guides/bugzilla.rst

```diff
@@ -0,0 +1,192 @@
+        # check that it exists on provider
+        if not virtualcenter_provider.mgmt.does_vm_exist(vm_name):
+            pytest.skip("No capandu_vm available on virtualcenter_provider of name {}".format(vm_name))
+
+        vms = [all_vms.pop(all_vm_names.index(vm_name))]
+
+        # do not run the policy simulation against more that 4 VMs
+        try:
+            vms.extend(all_vms[0:min(random.randint(1, len(all_vms)), 4)])
+        except ValueError:
+            pytest.skip("No other vms exist on provider to run policy simulation against.")
+
+        filtered_collection = collection.filter({"names": [vm.name for vm in vms]})
+        # Now perform the policy simulation
+        view = navigate_to(filtered_collection, "PolicySimulation")
+        # Select the correct policy profile
+        view.fill({"form": {"policy_profile": "{}".format(vm_compliance_policy_profile.description)}})
+
+        # Now check each quadicon and ensure that only cu-24x7 is compliant
+        for entity in view.form.entities.get_all():
+            state = entity.data["quad"]["bottomRight"]["tooltip"]
+            if entity.name == vm_name:
+                assert state == "Policy simulation successful."
+            else:
+                assert state == "Policy simulation failed with: false"
+
+This a nice test case because it combines several of the things above. It is `blocked`
+by the BZ 1717483, but it is providing automated test coverage for BZ 1711352. Note that it isn't
+providing coverage for BZ 1717483, so that BZ must be blocking a setup or teardown step of
+the test case. It also makes use of the ``Bugzilla`` docblock, discussed below.
+You can then run the following command to set qe_test_coverage to ``+`` for BZ 1711352.
+
+``miq bz coverage --set cfme/tests/control/test_bugs.py``
+
+A dry run of this command (i.e. without ``--set``) produces the following output:
+
+.. code-block:: console
+
+    The following BZs should have qe_test_coverage set to '+':
+        id: 1155284, qe_test_coverage: ?
+        id: 1243357, qe_test_coverage: ?
+        id: 1711352, qe_test_coverage: -
+
+Bugzilla docblock
+^^^^^^^^^^^^^^^^^
+The ``Bugzilla`` docblock is for listing any BZ's that are tangentially related to a test case.
+Blocker, automates, and coverage BZ's should be listed here. Any BZ related to a test case
+should also be put here. For example
+
+
+.. code-block:: python
+
+    @pytest.mark.meta(blockers=[BZ(1234561), automates=[1234567])
+    def test_automated():
+      """
+      This test case is blocked by BZ 1234561, but provides automated test coverage for
+      BZ 1234567. The following BZs are related to this test case:
+
+      Bugzilla:
+          1234561
+          1234567
+          1234562
+      """
+      assert True
+
+So here in the ``Bugzilla`` docblock, in addition to the two BZ's listed in the test case's
+metadata, there is an additional ``BZ 1234562`` which is in some way related to the test
```

Comment:
I'm not a 100% sure if we would want to encourage putting bzs that are some way related. I used to imagine that this list would really only contain BZs that are directly being tested by the test case. I'd like to know thoughts from others on this topic. Thanks.

Reply:
Usually, along with the main BZ, I also add any BZ that is remotely related to the test case in the Bugzilla block. I think it's a good way to keep track of them. It helps me when I'm trying to understand a test case, and I think it would also be helpful to other people who're trying to do the same.

Reply:
Yeah I was intentionally vague with my wording because it really is up to the contributor's discretion. I don't really think there are any hard and fast rules here. If the FA owner thinks it is useful to have the BZ listed there, i.e. it could explain a failing test, or help in debugging an unexpected result, then it should be put here.